custom-vu/src/main/kotlin/jces1209/vu/page/issue/AbstractProjectIssueNavigatorPage.kt

```diff
@@ -0,0 +1,5 @@
+package jces1209.vu.page.issue
+
+interface AbstractProjectIssueNavigatorPage {
```

Comment:
We don't have to call out the "abstract" part. This will do just fine:
```suggestion
interface ProjectIssueNavigatorPage {
```
`AbstractIssuePage` is called that way only because there already exists an `IssuePage`: `com.atlassian.performance.tools.jiraactions.api.page.IssuePage`.